internal/testcommon/k8s/telemetrygen.go

```diff
@@ -1,7 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package k8stest // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8stest"
+package k8s // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8stest"
```

Comment:
I don't have strong opinions, but I thought it was nice it was called `k8stest` as it was very clear this is "test stuff". With just `k8s` it's now not so clear. But then this is used in test code so.. maybe we gather the opinions of the others :)

Reply:
You're right - At first i thought it might be redundant since this is part of the `testcommon` module but having it in the package name itself does make it clearer to indicate that this is only for tests - will revert it to the previous name :)